lib/oppen/printer.rb

```diff
@@ -227,13 +229,14 @@ def check_stream
     #
     # @return [Nil]
     def advance_right
-      @right = (right + 1) % scan_stack.length
+      @right = (right + 1) % @size.length
       return if right != left
 
       raise 'Token queue full' if !config.upsize_stack?
 
-      @size, = Utils.upsize_circular_array(@size, @left)
-      @tokens, @left, @right = Utils.upsize_circular_array(@tokens, @left)
+      @scan_stack.update_indexes(-@left)
+      @size, = ScanStack.upsize_circular_array(@size, @left)
+      @tokens, @left, @right = ScanStack.upsize_circular_array(@tokens, @left)
```

Comment:
`size, =` is always bad style for us. Actually, if you can find the rubocop rule that disallows this style it would be great.

I always want something like `size, _` or `size, _ignored_val` … using `_` as a prefix to denote ignored things, and never an empty string.